### PR TITLE
First draft of line manager promotion guidance

### PR DESCRIPTION
**Pull request by StevenLeighton21**
This guidance doc is intended to help line managers understand what is involved in seeking a promotion, so they can better guide their reportees in how they can advance their careers at Made Tech.

### CODE REVIEW

File: guides/line-management/promotions.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+# Promotions
+
+This guidances is intended to help line managers to facilitate the career progression of the people they line manage.
```
Comment (kjdchapman) [inline guides/line-management/promotions.md:3]: There's some changes we could make here to make this read clearer, I think:
- using simpler words where possible, e.g. facilitate -> help, utilise -> use
- writing to the reader directly (using "you" for the line manager reading the doc)
- using smaller sentences

There's a few examples throughout the doc, but as an example for this line:
```suggestion
This guidance is written for line managers. It'll help you understand how promotions work at Made Tech. 

You're responsible for supporting people you line manage when they want to get promoted.
```

File: guides/line-management/promotions.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+## When should I look to promote someone?
+
+It's difficult to provide a one-size-fits-all answer to this question, but ultimately this decision is made when both you and the individual you line manage feel that individual is ready to take on the role they're to be promoted to in an active delivery team. A not exhaustive list of signs an individual may be ready includes:
+
+- They exhibit some or all of the core competencies for their desired role
+- They are consistently evidencing some or all attributes that are described in the SFIA level that corresponds to their desired role
+- They have examples of exemplory work or feedback received
```
Comment (kjdchapman) [inline guides/line-management/promotions.md:11]: nit: spelling
```suggestion
- They have examples of exemplary work or feedback received
```

**Comment by kjdchapman** [comment]
I have similar suggestions for other sections, but it feels unkind/unhelpful to just start re-writing things! What do you think the best way to go from here?

**Comment by StevenLeighton21** [comment]
> I have similar suggestions for other sections, but it feels unkind/unhelpful to just start re-writing things! What do you think the best way to go from here?

I've made a few tweaks in the same vein, I have no problem with you rewriting things :)

**Review by kjdchapman — APPROVED** [review]
As discussed, elsewhere, better to :shipit: and edit later rather than hold it up. :rocket:

**Pull request merged**